src/http/one/Tokenizer.cc

```diff
@@ -80,8 +81,7 @@ Http::One::Tokenizer::qdText(SBuf &returnedToken, const bool http1p0)
             SBuf escaped;
             if (!prefix(escaped, qPairChars, 1)) {
                 returnedToken.clear();
-                restoreLastCheckpoint();
-                return false;
+                throw TexcHere("invalid escaped characters");
```

Comment:
Same concern here. Not finding quoted text warrants a false result, not an exception.